src/styles/block/block.scss

```diff
@@ -0,0 +1,28 @@
+.block {
+  display: grid;
+  grid-template-rows: repeat(2, minmax($box-size, 1fr));
+  grid-template-columns: repeat(auto-fill, minmax($box-size, 1fr));
+
+  &__box {
+    text-align: center;
+    line-height: $box-size;
+  }
+  
+  @for $index from 1 through 6 {
+    &__box:nth-child(#{$index})::before {
+     content: "#{$index}";
+    }
+  }
```

Comment:
Nice use of ::before) I just added numbers in html code, lazy me))

Reply:
> Nice use of ::before) I just added numbers in html code, lazy me))

The good programmer is the lazy one. So you have chosen the right path)